### PR TITLE
Upgrade to new module syntax

### PR DESCRIPTION
**Pull request by agu-z**
Runs `roc format` on everything to upgrade to the new syntax. 

I'm basing this on `file-str-api` as @rtfeldman suggested. **Please consult him before merging**.

### CODE REVIEW

File: platform/Cmd.roc [hunk 1]
```diff
@@ -164,8 +162,8 @@ status = \@Cmd cmd ->
 exec : Str, List Str -> Task {} [CmdError Err]
 exec = \program, arguments ->
 
-    @Cmd cmd = new program |> args arguments
+    (@Cmd cmd) = new program |> args arguments
```
Comment (rtfeldman) [inline platform/Cmd.roc:165]: Was this a formatter change, or do we need this syntactically now?

Comment (agu-z) [reply platform/Cmd.roc:165]: We do not need the parens for this file to `check`. The formatter adds these both in the latest and previous nightly.

**Comment by rtfeldman** [comment]
@agu-z FYI I based https://github.com/roc-lang/basic-cli/pull/201 on this branch!

**Comment by Anton-4** [comment]
I'll take a look at these test failures

File: ci/all_tests.sh [hunk 2]
```diff
@@ -124,8 +124,5 @@ find . -type d -name "roc_nightly" -prune -o -type f -name "*.roc" -print | whil
     fi
 done
 
-# just build this until we fix it
-$ROC build ./ci/file-testBROKEN.roc $ROC_BUILD_FLAGS
```
Comment (Anton-4) [inline ci/all_tests.sh:128]: This was hitting `Option::unwrap() on a None value`

**Comment by Anton-4** [comment]
Can we merge this @rtfeldman?

**Pull request merged**